test/extended/deployments/deployments.go

```diff
@@ -551,4 +554,50 @@ var _ = g.Describe("deploymentconfigs", func() {
 			}
 		})
 	})
+
+	g.Describe("with minimum ready seconds set", func() {
+		g.It("should include various info in status [Conformance]", func() {
+			_, name, err := createFixture(oc, minReadySecondsFixture)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("verifying the deployment is marked running")
+			o.Expect(waitForLatestCondition(oc, name, deploymentRunTimeout, deploymentRunning)).NotTo(o.HaveOccurred())
+
+			g.By("verifying that all pods are ready")
+			config, err := oc.REST().DeploymentConfigs(oc.Namespace()).Get(name)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			selector := labels.Set(config.Spec.Selector).AsSelector()
+			opts := kapi.ListOptions{LabelSelector: selector}
+			ready := 0
+			if err := wait.PollImmediate(500*time.Millisecond, 1*time.Minute, func() (bool, error) {
+				pods, err := oc.KubeREST().Pods(oc.Namespace()).List(opts)
+				if err != nil {
+					return false, nil
+				}
+
+				ready = 0
+				for i := range pods.Items {
+					pod := pods.Items[i]
+					if kapi.IsPodReady(&pod) {
+						ready++
+					}
+				}
+
+				return len(pods.Items) == ready, nil
+			}); err != nil {
```

Comment:
`err` is a WaitTimeoutError ("timedout waiting for the condition") so in case it is non-nil I need to override with a more meaningful error.